pkg/config/config.go

```diff
@@ -223,6 +224,7 @@ type RoomConfig struct {
 	MaxMetadataSize    uint32             `yaml:"max_metadata_size,omitempty"`
 	PlayoutDelay       PlayoutDelayConfig `yaml:"playout_delay,omitempty"`
 	SyncStreams        bool               `yaml:"sync_streams,omitempty"`
+	EnablePSRPC        bool               `yaml:"use_psrpc,omitempty"`
```

Comment:
can this be contained within PSRPCConfig ?

Reply:
i put this here because this config is already passed into roomservice and we'll remove it eventually... it's also specific to roomservice so this seemed like an appropriate place.